src/reducers/fileTree.tsx

```diff
@@ -43,19 +43,19 @@ const recursiveSortInPlace = (node: DirectoryNode): void => {
   });
 };
 
-const getVersionName = (version: Version) => {
-  return getLocalizedString(version.addon.name);
+const getVersionName = (versionAddonName: LocalizedStringMap) => {
+  return getLocalizedString(versionAddonName);
 };
 
-export const getRootPath = (version: Version) => {
-  return `root-${getVersionName(version)}`;
+export const getRootPath = (versionAddonName: LocalizedStringMap) => {
+  return `root-${getVersionName(versionAddonName)}`;
```

Comment:
Why is `getVersionName()` kept here?

Reply:
I want to be able to call `getRootPath` without having to pass in a complete `Version` object, because we're now calling it from `createInternalVersion`.

Reply:
I see. Could you inline the content of `getVersionName` here? And get rid of `getVersionName` entirely?

Reply:
I do also call it in `buildFileTreeNodes`, so I'd have to inline the logic twice. That's why it exists.

Reply:
The `getVersionName` function should be renamed to `getAddonName` if you want to keep it, but it's confusing because this function will receive the addon's name, so calling `getLocalizedString` directly seems to make more sense to me.

Reply:
I think I'd rather rename it to `getAddonName` than have to call `getLocalizedString` in two different places. I'll do that.